src/Microsoft.ApplicationInsights.AspNet/Extensions/ApplicationInsightsServiceOptionsExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Microsoft.ApplicationInsights.AspNet.Extensions
+{
+    /// <summary>
+    /// Extensions for Application Insights Service Options that returns specific set of properties if serviceOptions is enabled. If not, it returns the default initial value.
+    /// </summary>
+    internal static class ApplicationInsightsServiceOptionsExtensions
+    {
+        private static ApplicationInsightsServiceOptions _serviceOptions = new ApplicationInsightsServiceOptions();
+
+        /// <summary>
+        /// Checks if sampling is disabled through service options.
+        /// </summary>
+        internal static bool GetDisableDefaultSampling(this ApplicationInsightsServiceOptions serviceOptions)
```

Comment:
The reason we add this, is we can have the functionality implementation separate from the definition. For example, tomorrow we have initializers as optional features. We add the respective list in the Options. We then access these extensions to do the functionality that we intended to do. We pass the service options and services and we keep implementation abstract from options. We don't want to expose any of those details to customers. That is why, we also ignore any null checks, as the implementation details is taken care in this extension.